AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/Sync/RemoteSyncEngine.swift

```diff
@@ -287,7 +288,8 @@ class RemoteSyncEngine: RemoteSyncEngineBehavior {
         syncEventEmitter = SyncEventEmitter(initialSyncOrchestrator: initialSyncOrchestrator,
                                             reconciliationQueue: reconciliationQueue)
 
-        readyEventEmitter = ReadyEventEmitter(remoteSyncEnginePublisher: publisher)
+        readyEventEmitter = ReadyEventEmitter(remoteSyncEnginePublisher: publisher,
+                                              callBack: { self.cancelEmitter() })
```

Comment:
The `callBack` I added in ReadyEventEmitter is returned here so that it calls the new method `cancelEmitter()`